main.go

```diff
@@ -407,6 +407,15 @@ func setupReconcilersAndWebhooks(ctx context.Context, mgr ctrl.Manager,
 		}
 	}
 
+	setupLog.Debug("enabling AWSMachineTemplate controller")
```

Comment:
This should be in the same scope as the `AWSMachine` controller so that it's included in the same block.....as we have flags to disable certain controllers.

Reply:
updated.